web-pages/whats-new.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>What's New</title>
+ <meta charset="utf-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+ <link href="css/bootstrap.min.css" rel="stylesheet">
+ <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"/>
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+ <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+ <link href="css/style-whatsnew.css" rel="stylesheet">

```

Comment:
You need to have the css linked correctly. The name of the file was style-whats-new.css, you are missing a hyphen.

Reply:
I fixed the css linking to the correct file.